local.tf

```diff
@@ -88,7 +88,7 @@ locals {
     metadata_http_put_response_hop_limit = null       # The desired HTTP PUT response hop limit for instance metadata requests.
     # Settings for launch templates with mixed instances policy
     override_instance_types                  = ["m5.large", "m5a.large", "m5d.large", "m5ad.large"] # A list of override instance types for mixed instances policy
-    on_demand_allocation_strategy            = null                                                 # Strategy to use when launching on-demand instances. Valid values: prioritized.
+    on_demand_allocation_strategy            = "prioritized"                                        # Strategy to use when launching on-demand instances. Valid values: prioritized.
```

Comment:
This change causes `dynamic "mixed_instances_policy"`'s `for_each` to activate; resulting in unconditionally enabling mixed instances.

Reply:
fixed :heavy_check_mark: